build.gradle.kts

```diff
@@ -1,62 +1,13 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
 plugins {
-  java
-  kotlin("jvm") version "1.3.61"
-}
-
-group = "com.mattprecious"
-version = "1.0-SNAPSHOT"
-
-repositories {
-  mavenCentral()
-}
-
-dependencies {
-  implementation(kotlin("stdlib-jdk8"))
-  implementation("com.squareup.okio", "okio", "2.4.2")
-  implementation("com.squareup.wire", "wire-runtime", "3.0.2")
-
-  testCompile("junit", "junit", "4.13-rc-1")
-}
-
-configure<JavaPluginConvention> {
-  sourceCompatibility = JavaVersion.VERSION_1_8
-}
-tasks.withType<KotlinCompile> {
-  kotlinOptions.jvmTarget = "1.8"
+  base
+  kotlin("jvm") version "1.3.61" apply false
```

Comment:
What does the apply do?

Reply:
It just sets the version in the root so it can be applied in the leafs.